src/modules/host/host.scss

```diff
@@ -0,0 +1,24 @@
+.tendrl-host-container {
+	.list-view-pf-additional-info {
+        float: none;
+        width: 100%;
+        margin: auto 1rem;
+        justify-content: space-between;
+    }
+    .list-view-pf .list-group-item-heading {
+        flex: 0 0 auto;
+    }
+    .list-view-pf-additional-info-item {
+        flex-direction: column;
+    }
+    .list-view-pf-additional-info-item strong {
+        margin: auto 5px;
+    }
+    .list-view-pf-additional-info-item.deflex {
+        flex-basis: 10%;
+    }
+    .focus {
+        font-weight: 600;
+        font-size: 2rem;
+    }
+}
```

Comment:
cluster.scss and host.scss has common content. Can they be extracted and reused?